Source/AliveLibAO/Abe.cpp

```diff
@@ -2738,11 +2740,393 @@ void Abe::VOn_Tlv_Collision_421130(Path_TLV *pTlv)
     }
 }
 
-void Abe::State_0_Idle_423520()
+void Abe::TryHoist_423420()
 {
     NOT_IMPLEMENTED();
 }
 
+__int16 Abe::HandleDoAction_429A70()
+{
+    NOT_IMPLEMENTED();
+    return 0;
+}
+
+static bool IsSameScaleAsHoist(Path_Hoist* pHoist, BaseAliveGameObject* pObj)
+{
+    auto width = pHoist->field_14_bottom_right.field_0_x - pHoist->field_10_top_left.field_0_x;
+
+    if (pObj->field_BC_sprite_scale == FP_FromInteger(1))
```

Comment:
I  think this is worth a hack comment saying the width is used to determine the layer as there is no layer property in the TLV

Reply:
added